dataservice/api/read_group/models.py

```diff
@@ -43,20 +48,20 @@ class ReadGroupGenomicFile(db.Model, Base):
     __tablename__ = 'read_group_genomic_file'
     __prefix__ = 'RF'
     __table_args__ = (db.UniqueConstraint('read_group_id',
-                                          'genomic_file_id'),)
+                                          'genomic_file_id',),)
     read_group_id = db.Column(KfId(),
                               db.ForeignKey('read_group.kf_id'),
                               nullable=False)
 
     genomic_file_id = db.Column(KfId(),
                                 db.ForeignKey('genomic_file.kf_id'),
                                 nullable=False)
-    read_group = db.relationship('ReadGroup')
-    genomic_file = db.relationship('GenomicFile')
+    external_id = db.Column(db.Text(),
```

Comment:
Do we have any of these?

Reply:
This is super useful during ingest debugging and dev because you can set the external_id of a read_group_genomic_file to be the read_group external_id concatenated with the genomic_file external_id. I do this for pretty much any entity representing an association table. Its so nice to be able to query the entity and see the external_ids in addition to the KF IDs in the results.

Reply:
We should stop abusing "external ID" for internal debugging values. Maybe add another generic field to all entities.

Reply:
Why? Its where I actually do store the contributor's ID for an entity, and I do use that to help debug/verify that all of my entities got into the dataservice

Reply:
Maybe we should just add external_id to the base model so everything has it?

Reply:
external_id should be in the base model, but also kf_synthetic_id should be there too. Using the same field for both muddies the meaning. If they don't provide their own id, us filling in that field with something else is deceptive